src/multi.jl

```diff
@@ -76,7 +162,7 @@ end
 
 """
     TODO: docstring
-    method for monomial
+    extreme value calculation method for monomial (simple)
```

Comment:
What is the actual relaxation being applied? 
Is it the relaxation in the JOGO paper?

Reply:
Yes. Here, it is just the extreme point evaluation, which is exactly calculating b1^2 b2^2 ... bn^2.